### PR TITLE
Build fluent-bit container image.

### PR DESCRIPTION
**Pull request by d-kuro**
refs: https://github.com/cybozu-go/moco/issues/535

Push fluent-bit container image to `ghcr.io`.

This PR only pushes the image.
The pushed image needs to be changed to be used by MOCO, but this will be done in another PR.

GitHub Actions workflow was tested in a forked repository.
https://github.com/d-kuro/moco/pkgs/container/moco%2Ffluent-bit

### CODE REVIEW

**Review by masa213f — CHANGES_REQUESTED** [review]

File: .github/workflows/build-fluent-bit-container.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          fi
+          ./tag_branch_consistency fluent-bit
+
+      - uses: docker/build-push-action@v4
+        if: github.ref == 'refs/heads/main'
+        with:
+          context: containers/fluent-bit/.
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: ghcr.io/cybozu-go/moco/fluent-bit:${{ env.TAG }}
```
[the remark this answers is not in the document]
Comment (masa213f) [reply .github/workflows/build-fluent-bit-container.yaml:54]: Sorry, the BRANCH tag is not used. It need not push the BRANCH tag.

Please delete the `BRANCH` file for fluent-bit.

File: containers/fluent-bit/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Fluent Bit container
+====================
+
+Build Docker container image for [Fluent Bit][], Log Processor and Forwarder.
+
+Docker images
+-------------
+
+Docker images are available on [Quay.io](https://quay.io/repository/cybozu/fluent-bit)
```
[the remark this answers is not in the document]
Comment (d-kuro) [reply containers/fluent-bit/README.md]: Fixed: https://github.com/cybozu-go/moco/pull/528/commits/6227ff09880bdbd2551a1d80ccc37862a23be918

File: containers/tag_exists [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+else
+    DIR="$NAME"
+fi
+TAG=$(cat "$DIR"/TAG)
+
+TOKEN=$(curl -sSfL https://ghcr.io/token\?scope="repository:cybozu-go/moco:pull" | jq -r .token)
+HTTP_STATUS=$(curl --head -H "Authorization: Bearer $TOKEN" -sSL -w "%{http_code}" "https://ghcr.io/v2/cybozu-go/moco/$NAME/manifests/$TAG" -o /dev/null)
+
+if [ "$HTTP_STATUS" = "404" ]; then
+    echo "ok"
```
[the remark this answers is not in the document]
Comment (d-kuro) [reply containers/tag_exists]: Thank you very much. Other bugs existed in the script, so I rewrote it.

https://github.com/cybozu-go/moco/pull/528/commits/875ff68ca69dc9799fa28405e95f14c48dfeb321

File: .github/workflows/build-fluent-bit-container.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (d-kuro) [reply .github/workflows/build-fluent-bit-container.yaml:46]: Remove it: https://github.com/cybozu-go/moco/pull/528/commits/e0b3475514bd760b967a56783dd83407923890c6

File: containers/tag_exists [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+NAME="$1"
+if [ $# -eq 2 ]; then
+    DIR="$2"
+else
+    DIR="$NAME"
+fi
+TAG=$(cat "$DIR"/TAG)
+
+TOKEN=$(curl -sSfL https://ghcr.io/token\?scope="repository:cybozu-go/moco:pull" | jq -r .token)
+HTTP_STATUS=$(curl --head -H "Authorization: Bearer $TOKEN" -sSL -w "%{http_code}" "https://ghcr.io/v2/cybozu-go/moco/$NAME/manifests/$TAG" -o /dev/null)
```
[the remark this answers is not in the document]
Comment (d-kuro) [reply containers/tag_exists]: Thanks!

https://github.com/cybozu-go/moco/pull/528/commits/21889fd8d4a6badbed4e36cab758536290cbc357

**Review by masa213f — CHANGES_REQUESTED** [review]

**Comment by d-kuro** [comment]
@masa213f 
Thanks for the review.
Please review again as I have made the corrections in the following commit 🙏 

https://github.com/cybozu-go/moco/pull/528/commits/aa5c8356c736bb345c5138175099996a21f471e2

**Review by masa213f — APPROVED** [review]
LGTM

**Pull request merged**